### ISSUE TITLE
artistic license not a default on github

### ISSUE THREAD

**Issue by coke** [Raku/problem-solving#353]
When creating a new repository on github, you have an option to select a license from a drop down. Artistic License 2.0 isn't one of those options.

Someone should submit a request to GH to add the most common Raku module license.

**Comment by JJ** [comment]
It's simply not possible to do that, or if it were, we would have no guarantee that it would be accepted.
What GitHub does is to use license description from [SPDX, described here](https://github.com/spdx/license-list-XML). For instance, [this change to the Artistic license you're mentioning](https://github.com/spdx/license-list-XML/pull/1167).
That's the extent of it, however. Whatever GitHub uses to pick a few "default" licenses, it's their own policy.
At any rate, this would be something for The Perl and Raku foundation, not the Raku community.

**Comment by coke** [comment]
I'm confused - it's not possible to... open a ticket? (I get we don't control it, it's their policy, and they might reject it - but they won't consider it at all if we don't ask them.)

"Change you're mentioning" - I didn't mention this change, not sure what you're referring to;

The SPDX list does include artistic 2.0 ... this is just to highlight that the decision of what to include is github's, yes, which is who *someone* should ask to update the list.

**Comment by JJ** [comment]
> I'm confused - it's not possible to... open a ticket? (I get we don't control it, it's their policy, and they might reject it - but they won't consider it at all if we don't ask them.)

You can most definitely  ask customer support. Or open a discussion in github.community. That last one will at least have the nice side effect of being public. However, I don't think that will buy you anything. Or that you will need community endorsement or support for doing any of that.
> 
> "Change you're mentioning" - I didn't mention this change, not sure what you're referring to;
> 
> The SPDX list does include artistic 2.0 ... is this just to highlight that the decision of what to include is github's?

It does, and after the PR, with a correct link to its definition. However, what you were mentioning is which ones of these available (and recognizable) licenses are available by default; only a few of them are available, I guess by choice of some product manager. It's not, as some other features (the documentation, the heuristics to recognize languages, or the gitignore files, to name the ones I've in the past interacted with) open to community suggestions, or requests, as far as I can tell.

Even if there was a way of doing that, Raku stuff is not the one that majoritarily uses Artistic -- Perl is. So if there's some petitioning happening, it would probably have some real heft if it's done under the umbrella organization that covers us both... Not us.

Long story short: it's not a problem, but if it is, it's not *our* problem (as in ours and no one else's): it's TPRF's.

BTW, the only thing that dropdown menu does is to create a LICENSE file with the correct content. You can achieve the same with any of the existing Raku distro templates (I created one, there must be others) or with any of the generators. Unfortunately, if you try to "add" a license from the "community standards" page, it's not there either. I guess it's not popular enough for that, or who knows. But, again, it's a product decision which is apparently closed to our community.
